app/src/test/java/uc/edu/ozturkkl/traveltracker/LocationUnitTest.kt

```diff
@@ -13,11 +13,12 @@ class LocationUnitTest {
 
     lateinit var mvm: MainViewModel
 
-    //this test will fail because I haven't added the API key directly into the code for security reasons
-//    @Test
-//    fun canFindAddress() {
-//        mvm = MainViewModel()
-//        mvm.loadPredictions("2600 Clifton Avenue");
-//        assertTrue(mvm.locations.contains("2600 Clifton Avenue, Cincinnati, OH, USA"))
-//    }
+    /*this test will fail because I haven't added the API key directly into the code for security reasons
```

Comment:
Remove commented out code entirely.  Instead of commenting out code, my suggestion is:
1) Use feature branches in Git to keep experimental work separate from master.
2) If you're removing something that used to be there, just remove it entirely.  You can always go back in history to see what used to be there.